quickstart/csharp/dotnetcore/from-microphone/helloworld/Program.cs

```diff
@@ -3,64 +3,51 @@
 // Licensed under the MIT license. See LICENSE.md file in the project root for full license information.
 //
 
-// <code>
+// <stt-quickstart>
```

Comment:
I was thinking maybe all our tags should start with a common prefix such as "code-embedded-in-microsoft-docs-..." or something of the sort, so both we (Carbon devs) and external developers know what these are for. Maybe discussed in the next meeting (but I'm out on Monday). Don't let that block this check in, just something to discuss. I think that would be helpful.